src/Silverstripe/PingdomCLI/GetChecks.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace Silverstripe\PingdomCLI;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputArgument;
+use Silverstripe\Pingdom\Api;
+use Symfony\Component\Console\Helper\Table;
+
+class GetChecks extends Command
+{
+    // the name of the command (the part after "bin/console")
+    protected static $defaultName = 'get-checks';
```

Comment:
Our `symfony/console` convention would mean `checks:get` is a better approach. That way we can add future `checks:*` commands under one command namespace